lldb/unittests/Core/TelemetryTest.cpp

```diff
@@ -81,6 +81,20 @@ using namespace lldb_private::telemetry;
 class TelemetryTest : public testing::Test {
 public:
   lldb_private::SubsystemRAII<lldb_private::FakePlugin> subsystems;
+  std::vector<std::unique_ptr<::llvm::telemetry::TelemetryInfo>>
+      received_entries;
+
+  void SetUp() override {
+    // This would have been called by the plugin reg in a "real" pluging
+    // For tests, we just call it directly.
+    lldb_private::FakePlugin::Initialize();
```

Comment:
I don't think this is needed anymore with the `SubsystemRAII` above?

Reply:
done